Gopkg.toml

```diff
@@ -2,14 +2,18 @@
   name = "k8s.io/utils"
   branch = "master"
 
-[[constraint]]
-  name = "k8s.io/apimachinery"
-  branch = "master"
-
 [[constraint]]
   name = "gopkg.in/yaml.v2"
   branch = "v2"
 
 [[constraint]]
   name = "github.com/oracle/oci-go-sdk"
   version = "v1.0.0"
+
+[[constraint]]
+  branch = "release-1.11"
```

Comment:
Approval needed.

Reply:
We've generally moved from using the release branch to specifying specific versions of kube deps. We should already have pre-approvals of specific 1.11.x versions of apimachinery I believe too 🙂

Reply:
Approved for kubernetes-1.11.1